modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -469,8 +469,12 @@ export default function EditNotificationTemplate({
 														},
 													})
 												}
```

Comment:
H @danvb112. I'm sending this code to make an adjustment in a previous PR that [Brian asked me to do](https://github.com/brianchandotcom/liferay-portal/pull/124638).
Thanks.